.docker/nginx.conf

```diff
@@ -0,0 +1,57 @@
+# Upstream to abstract backend connection(s) for PHP
+upstream php {
+        server unix:/tmp/php-cgi.socket;
+        server 127.0.0.1:9000;
+}
+
+server {
+    listen 80 default_server;
```

Comment:
So this configuration will actually go into `docker-compose.yml:11` where you can choose a different port on your host machine to map to port 80 of Docker's webserver. For egs, we map host's 3307 to MySQL Docker's 3306 to avoid conflict. 

That said, using any port other than 80 might not work because lots of image path are hardcoded without port number in the database. So we would have to clean that first, which should ideally be a different PR.

You can comment out the line in `docker-compose.yml` in your local setup if docker-setup conflicts with your local one.

Reply:
🤔 

Yeah, let's just go with this for now, and I can play around with it later and maybe iterate.

I wonder if there's a way to do an "local" config that overrides parts of `docker-compose.yml`, similar to how you can have `phpcs.xml.dist` committed to version control, and then an ignored `phpcs.xml` file that takes precedence.